src/utils/validations.js

```diff
@@ -112,19 +112,15 @@ const validateTripRequest = async (req, res, next) => {
   }
   const whiteSpaces = /\s+/g;
   travelType = String(travelType).replace(whiteSpaces, ' ').trim().toLowerCase();
-  switch (travelType) {
-    case 'one-way':
-      try {
-        const validationOutput = await validateOneWayTripRequest(tripRequestInfo);
-        validationOutput.userId = req.sessionUser.id;
-        req.body = validationOutput;
-        return next();
-      } catch (validationError) {
-        return errorResponse(res, badRequest, validationError.message);
-      }
-    default:
-      return errorResponse(res, badRequest, invalidTravelType);
-  }
+    try {
+      const validationOutput = travelType === 'one-way' ? await validateOneWayTripRequest(tripRequestInfo)
+      : await validateMultiCitiesTripRequest(tripRequestInfo);
+      validationOutput.userId = req.sessionUser.id;
+      req.body = validationOutput;
+      return next();
+    } catch (validationError) {
+       return errorResponse(res, badRequest, validationError.message);
+    }
```

Comment:
why did you delete the switch case here, you could have created your own case block, and added your code there, also remember that 'return-trip' will use it too.